tests/4.5/target_teams_distribute/test_target_teams_distribute_default_none.F90

```diff
@@ -0,0 +1,87 @@
+!===--- test_target_teams_distribute_default_none.F90-----------------------===//
+!
+! OpenMP API Version 4.5 Nov 2015
+!
+! This tests uses the default(none) clause on a target teams distribute test.
+! The test aims to validate that all values will not have default data sharing
+! attributes.
+!
+!===------------------------------------------------------------------------===//
+
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_teams_distribute_default_none
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_WARNING("Test only uses default(none) clause and does not")
+  OMPVV_WARNING("guarantee that the default(none) is enforced.")
```

Comment:
if you want these to be on a single line it is really annoying. This is fine with me, but I used to do:
```
! Declare this
         CHARACTER(len=300) :: infoMessage
! Use it as storage for the whole message
WRITE(infoMessage, *) " working on a shared memory &
           & environment. Test might be inconclusive"
           OMPVV_WARNING(infoMessage)
```